src/ray/raylet/scheduling/scheduling_policy.cc

```diff
@@ -14,41 +14,76 @@
 
 #include "ray/raylet/scheduling/scheduling_policy.h"
 
+#include <functional>
+
 namespace ray {
 
 namespace raylet_scheduling_policy {
+namespace {
+bool IsGPURequest(const ResourceRequest &resource_request) {
+  if (resource_request.predefined_resources.size() <= GPU) {
+    return true;
+  }
+  return resource_request.predefined_resources[GPU] == 0;
```

Comment:
Should invert the logic here, currently this implements "IsNotGPURequest"

Reply:
done